beacon_node/network/src/sync/block_sidecar_coupling.rs

```diff
@@ -29,18 +28,13 @@ pub struct RangeBlockComponentsRequest<E: EthSpec> {
     /// Used to determine if the number of data columns stream termination this accumulator should
     /// wait for. This may be less than the number of `expects_custody_columns` due to request batching.
     num_custody_column_requests: Option<usize>,
-    /// The peers the request was made to.
-    pub(crate) peer_ids: Vec<PeerId>,
-    max_blobs_per_block: usize,
```

Comment:
Removed this from the state, since `into_responses_with_blobs` has access to the spec